src/utils/url.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org-cloudflare-worker for the canonical source repository
+ */
+import URL from 'core-js-pure/features/url'
+
+const UrlProperties = ['subdomain', 'hostname', 'pathname'] as const
+type UrlProperties = typeof UrlProperties[number]
+
+const contentApiParameters = [
+  'contentOnly',
+  'hideTopbar',
+  'hideLeftSidebar',
+  'hideRightSidebar',
+  'hideBreadcrumbs',
+  'hideDiscussions',
+  'hideBanner',
+  'hideHorizon',
+  'hideFooter',
+  'fullWidth',
+]
+
+export class Url extends URL {
+  get subdomain(): string {
+    return this.hostname.split('.').slice(0, -2).join('.')
+  }
```

Comment:
I changed the signature for the return type from `string | null` to `string` -> instead of `null` the empty is returned when there is no subdomain. Okay for you?